### PR TITLE
Fixed Flake8

### PR DESCRIPTION
**Pull request by GameDungeon**
## Overview: What does this pull request change?

Flake8 has been broken for some time now without anyone noticing. I noticed and then fixed it, and also fixed some of the accumulated errors.

## Reviewer Checklist
- [x] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [x] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [x] If applicable: newly added functions and classes are tested

### CODE REVIEW

File: manim/camera/moving_camera.py [hunk 1]
```diff
@@ -213,7 +213,7 @@ def auto_zoom(self, mobjects, margin=0, only_mobjects_in_frame=False):
                 continue
 
             # initialize scene critical points with first mobjects critical points
-            if scene_critical_x_left == None:
```
Comment (christopher-besch) [inline manim/camera/moving_camera.py:216]: That hurts.

Comment (GameDungeon) [reply manim/camera/moving_camera.py:216]: well I fixed it for you 😄

**Review by christopher-besch — APPROVED** [review]
This looks very good. I've tested it on my machine and it works the way it should.
I think we can merge this. Good job @GameDungeon!

**Review by Darylgolden — APPROVED** [review]
LGTM, thank you!

**Pull request merged**